.github/workflows/cicd_ts_main_on_push.yml

```diff
@@ -57,7 +57,6 @@ jobs:
         run: echo "SHORT_SHA=`echo ${GITHUB_SHA} | cut -c1-8`" >> $GITHUB_ENV
       - name: Publish to npm
         working-directory: ${{ env.working_dir }}
-        if: github.event_name == 'push'
```

Comment:
Why do we remove this ?

Reply:
because i need to build a version to test when it's in PR